tests/src/rules/order.js

```diff
@@ -2338,11 +2338,11 @@ context('TypeScript', function () {
           // Option alphabetize: {order: 'asc'}
           test({
             code: `
-              import c from 'Bar';
               import type { C } from 'Bar';
+              import c from 'Bar';
```

Comment:
Changing existing tests implies it’s a breaking change. Can this test be reverted and a new one added?